demo/data/veterans_lung_cancer-test.csv

```diff
@@ -0,0 +1,42 @@
+Survival_label_lower_bound,Survival_label_upper_bound,Age_in_years,Karnofsky_score,Months_from_Diagnosis,Celltype=adeno,Celltype=large,Celltype=smallcell,Celltype=squamous,Prior_therapy=no,Prior_therapy=yes,Treatment=standard,Treatment=test
```

Comment:
Not entirely sure if we need to make a copy of the data in order to have train-test split. Do we have a slice operator with flink data structure?

Reply:
no, we do not :( I can move this to the test resources for the smoke tests, though.

Reply:
Maybe we just use the training dataset for running tests for now, maybe with some added noise. We are kind of conservative about adding data files in git repository.

Reply:
No worried, I worked it around. It should be much better now.